WordPress/Docs/CodeAnalysis/EscapedNotTranslatedStandard.xml

```diff
@@ -0,0 +1,20 @@
+<documentation title="Escaped Not Translated Text">
+    <standard>
+    <![CDATA[
+    Text intended for translation needs to be wrapped in a localization function call.
+    This sniff will help you find instances where text is escaped for output, but no localization function is called.
```

Comment:
I'd like to see some sort of extra clarification in here re number of args - something like:

> This sniff will help you find instances where text is escaped for output, but not with a localization function, despite there appearing to be a text domain argument.

Reply:
I've adjusted the text along the lines of your suggestion. Better now ?